### ISSUE TITLE
List of Dead Links in Website

### ISSUE THREAD

**Issue by mhmohona** [MDAnalysis/MDAnalysis.github.io#405]
I checked [MDAnalysis Main Website](https://www.mdanalysis.org/) and found several links that are either broken, lead to 404 errors, or are inaccessible -

| **#** | **Error**          | **Anchor Text**                | **Linked From**                                               | **Broken Link**                                                                                  |
|-------|--------------------|--------------------------------|---------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
| 1     | -1 Timeout         | NumPy                          | [MDAnalysis Main Page](https://www.mdanalysis.org/)           | [https://numpy.scipy.org/](https://numpy.scipy.org/)                                             |
| 2     | 403 Forbidden      | img/src                        | [MDAnalysis Main Page](https://www.mdanalysis.org/)           | [https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67](https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67) |
| 3     | 404 Not Found      | @orioncohen                    | [About Page](https://www.mdanalysis.org/about/)               | [https://github.com/orioncohen](https://github.com/orioncohen)                                   |
| 4     | 404 Not Found      | nglviewer                      | [Learning Page](https://www.mdanalysis.org/pages/learning_MDAnalysis/) | [https://www.mdanalysis.org/pages/learning_MDAnalysis/nglviewer.org/nglview/latest/](https://www.mdanalysis.org/pages/learning_MDAnalysis/nglviewer.org/nglview/latest/) |
| 5     | 404 Not Found      | King’s College London          | [UGM 2024 Page](https://www.mdanalysis.org/pages/ugm2024/)    | [https://www.kcl.ac.uk/accommodation/summer-accommodation](https://www.kcl.ac.uk/accommodation/summer-accommodation) |
| 6     | -1 Not Found       | https://www.rokarestaurant.com/en | [UGM 2024 Page](https://www.mdanalysis.org/pages/ugm2024/) | [https://www.rokarestaurant.com/en](https://www.rokarestaurant.com/en)                          |
| 7     | 404 Not Found      | @richardgowers                 | [GSoC 2024 Post](https://www.mdanalysis.org/2024/08/30/gsoc2024-final-sm/) | [https://github.com/richardgowers](https://github.com/richardgowers)                             |
| 8     | 404 Not Found      | examples                       | [Errata Page](https://www.mdanalysis.org/pages/errata/)       | [https://github.com/MDAnalysis/mdanalysis/tree/master/package/examples](https://github.com/MDAnalysis/mdanalysis/tree/master/package/examples) |
| 9     | 404 Not Found      | examples/blocks.py             | [Errata Page](https://www.mdanalysis.org/pages/errata/)       | [https://github.com/MDAnalysis/mdanalysis/blob/master/package/examples/blocks.py](https://github.com/MDAnalysis/mdanalysis/blob/master/package/examples/blocks.py) |
| 10    | 404 Not Found      | examples/radial_distribution_function.py | [Errata Page](https://www.mdanalysis.org/pages/errata/) | [https://github.com/MDAnalysis/mdanalysis/blob/master/package/examples/radial_distribution_function.py](https://github.com/MDAnalysis/mdanalysis/blob/master/package/examples/radial_distribution_function.py) |
| 11    | -1 Timeout         | Radisson Blu Hotel             | [UGM 2023 Page](https://www.mdanalysis.org/pages/ugm2023/)    | [https://www.radissonhotels.com/en-us/hotels/radisson-blu-lisbon](https://www.radissonhotels.com/en-us/hotels/radisson-blu-lisbon) |
| 12    | 429 Too Many Requests | Hyatt Place Tempe           | [Blog Page](https://www.mdanalysis.org/blog/page4/)          | [https://www.hyatt.com/en-US/hotel/arizona/hyatt-place-tempe-phoenix-university/phxzt](https://www.hyatt.com/en-US/hotel/arizona/hyatt-place-tempe-phoenix-university/phxzt) |
| 13    | 404 Not Found      | here                           | [GSoC Blog Post](https://www.mdanalysis.org/2018/04/26/gsoc-students/) | [https://github.com/numfocus/gsoc/blob/master/2018/accepted_student_blogs.md](https://github.com/numfocus/gsoc/blob/master/2018/accepted_student_blogs.md) |
| 14    | 404 Not Found      | MDAnalysis Tutorial            | [GSOD 2019 Post](https://www.mdanalysis.org/2019/04/23/gsod2019/) | [https://www.mdanalysis.org/MDAnalysisTutorial/](https://www.mdanalysis.org/MDAnalysisTutorial/) |
| 15    | 403 Forbidden      | img/src                        | [SPIDAL Tutorial](https://www.mdanalysis.org/SPIDAL-MDAnalysis-Midas-tutorial/) | [https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png](https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png) |
| 16    | 404 Not Found      | mda-pymol                      | [Building Block Post](https://www.mdanalysis.org/2019/04/20/mdanalysis-as-a-building-block/) | [https://nms.kcl.ac.uk/lorenz.lab/wp/2019/04/17/mda-pymol/](https://nms.kcl.ac.uk/lorenz.lab/wp/2019/04/17/mda-pymol/) |
| 17    | -1 Not Found       | MMTF                           | [Licensing Update Post](https://www.mdanalysis.org/2023/09/22/licensing-update/) | [https://mmtf.rcsb.org/](https://mmtf.rcsb.org/)                                                |
| 18    | 404 Not Found      | HOLE2 MDAKit                   | [Blog Page](https://www.mdanalysis.org/blog/page9/)          | [https://www.mdanalysis.org/hole2-mdakit/](https://www.mdanalysis.org/hole2-mdakit/)             |

I am ready to start working on fixing these issues, but I would appreciate some feedback or suggestions before proceeding. Also, if there are preferred sources to replace outdated links, please provide them.

**Comment by RMeli** [comment]
Thanks @mhmohona. Please see my comment in https://github.com/MDAnalysis/UserGuide/issues/393#issuecomment-2414904321, which applies here too.

**Comment by p-j-smith** [comment]
Nice finds @mhmohona. Did you use [lychee](https://github.com/lycheeverse/lychee) to check the links? We could consider using the [lychee GitHub Action](https://github.com/lycheeverse/lychee-action) across our repos to ensure links are always up to date

**Comment by orbeckst** [comment]
I don't think that we need to fix links on blog pages, which are somewhat ephemeral anyway. I'd consider it more historically accurate if these deadlinks remained (eg MMTF) as this just documents the knowledge at the time.

Links elsewhere (anything under /pages and on the home page of course) should be fixed (with the exception of @RMeli 's https://github.com/MDAnalysis/MDAnalysis.github.io/issues/405#issuecomment-2428460223 ).
